src/main/java/com/ecommerce/service/priceService/PriceService.java

```diff
@@ -0,0 +1,13 @@
+package com.ecommerce.service.priceService;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface PriceService<Price>{
```

Comment:
I hope you understand why do we use generic types. This is not a correct place to use a generic.